core/contracts/financial-templates/implementation/demo/DepositBox.sol

```diff
@@ -0,0 +1,346 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+import "../FeePayer.sol";
+import "../../../common/implementation/FixedPoint.sol";
```

Comment:
small nit: to be consistent with your other line breaks this should have a break in between (line break for different import contexts)

Reply:
you mean between `FeePayer` and `../../../common/*` right?